src/components/ReviewAnswers.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Button, Card } from 'react-bootstrap';
+
+const getAnswersClass = (question, userAnswers, answer) => {
+  if (answer.isCorrect) {
+    return 'text-success';
+  } else if (userAnswers[question.id] === answer.id) {
+    return 'text-danger';
+  } else {
+    return '';
+  }
+};
+
+const ReviewAnswers = ({
+  score,
+  questions,
+  quizName,
+  userAnswers,
+  numOfQuestions,
+}) => {
+  return (
+    <div>
+      <div>
+        <h3>
+          Your score is : {score} / {numOfQuestions}
+        </h3>
+      </div>
+      {questions.map(question => (
+        <Card>
+          <Card.Body>
+            <Card.Title>{question.text}</Card.Title>
+            <Card.Text>
+              <ul>
```

Comment:
The `ul` element by default has some margin at the bottom which is adding a bit more space to the bottom of the card. Let's add `mb-0` to remove this extra space